man/systemd.netdev.xml

```diff
@@ -315,6 +315,25 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>AgeingTimeSec=</varname></term>
+          <listitem>
+            <para>AgeingTimeSec specifies the number of seconds a MAC Address will be kept in
+            the forwaring database after having a packet received from this MAC Address.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Priority=</varname></term>
+          <listitem>
+            <para>An unsigned. The priority of the bridge.</para>
```

Comment:
can you elaborate on the range of this? i.e. what the smallest and largest acceptable value is?